applicationinsights.ts

```diff
@@ -48,7 +48,7 @@ class ApplicationInsights {
      * @returns {ApplicationInsights/Client} a new client
      */
     public static getClient(instrumentationKey?: string) {
```

Comment:
It would be really great to rename this :smile:

Reply:
Ashamed to admit, but I don't get the instinctive gag reflex when typing `getClient` :) However I would happy to rename, any suggestions? Semantically `createClient` makes more sense, but what did you have in mind? Just `.client`?

Reply:
We have `.client` already, which does what `getClient` should do - return the current client. `createClient` or `newClient` would be a lot better. Alternatively we can just expose the `Client` class and users can use `new Client()`

Reply:
Got it, will change.